File: ComfortProfilesSharingFront/comfort-profiles-sharing/src/logic/comfort-profile.ts
import { ComfortProfileRoom } from "./comfort-profile-room";
import { CoffeeType } from "./coffee-type.model";
import { PreferableCoffeeTime } from "./pref-coffee-time.model";
import { PreferableTeaTimes } from "./pref-tea-time.model";

export class ComfortProfile{
    public userId : string;
    public shoeSize : number;
    public clothingSize : number;
    public chairTypeId : number;
    public tableTypeId : number;
    public mattressTypeId : number;
    public waterTypeId : number;
    public allergens : string;
    public kindOfTea : string;
    public kindOfCoffee : string;
    public musicalPreferences : string;
    public fruitPreferences : string;
    public comfortTeapotTemperature : number;
    public preferableRoomsIndicators : Array<ComfortProfileRoom>;
    public favoriteCoffeeTypes : Array<CoffeeType>;
    public preferableCoffeeTimes: Array<PreferableCoffeeTime>;
    public preferableTeaTimes: Array<PreferableTeaTimes>
}

File: CPSMobile/CPSMobile/CPSMobile/Models/ComfortPRofile.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace CPSMobile.Models
{
    public class ComfortProfile
    {
        public string userId { get; set; }
        public int shoeSize { get; set; }
        public int clothingSize { get; set; }
        public string allergens { get; set; }
        public string kindOfTea { get; set; }
        public string kindOfCoffee { get; set; }
        public string musicalPreferences { get; set; }
        public string fruitPreferences { get; set; }
        public int chairTypeId { get; set; }
        public int tableTypeId { get; set; }
        public int mattressTypeId { get; set; }
        public int waterTypeId { get; set; }

        public List<PreferableRoomsIndicator> preferableRoomsIndicators { get; set; }
        public List<FavoriteCoffeeType> favoriteCoffeeTypes { get; set; }
        public List<PreferableCoffeeTime> preferableCoffeeTimes { get; set; }
        public int comfortTeapotTemperature { get; set; }
        public List<PreferableTeaTime> preferableTeaTimes { get; set; }
    }
}
